client/src/components/FoodSeeker/SearchResults/layouts/Tablet.js

```diff
@@ -1,30 +1,14 @@
 import React from "react";
-import makeStyles from "@mui/styles/makeStyles";
-
-const useStyles = makeStyles((theme) => ({
-  container: {
-    height: "100%",
-    overflow: "hidden",
-  },
-  map: {
-    height: "50%",
-  },
-  list: {
-    height: "50%",
-    overflow: "auto",
-  },
-}));
+import { Box } from "@mui/material";
 
 const TabletLayout = ({ filters, list, map }) => {
-  const classes = useStyles();
-
   return (
     <>
       {filters}
-      <div className={classes.container}>
-        <div className={classes.map}>{map}</div>
-        <div className={classes.list}>{list}</div>
-      </div>
+      <Box sx={{ height: "100%", overflow: "hidden"}}>
+        <Box sx={{ height: "50%"}}>{map}</Box>
+        <Box sx={{ height: "50%", overflow: "hidden" }}>{list}</Box>
```

Comment:
Hi @jenny-alexander, thanks for working on this issue. Quick question: I noticed that you have changed the overflow property for list from "auto" to "hidden". Visually it looks the same. Is there a reason to prefer one over the other in this case? I will read up more on it but please let me know if you have come across anything already. Thank you.

Reply:
@SAUMILDHANKAR - thank you for catching this typo! I fixed it. Can you review the change when you are able?

Reply:
Rebase done!